src/rgw/rgw_sal_motr.cc

```diff
@@ -232,32 +232,16 @@ int MotrUser::create_bucket(const DoutPrefixProvider* dpp,
 
   if (ret != -ENOENT) {
     *existed = true;
-    if (swift_ver_location.empty()) {
```

Comment:
Let this section as is  .. may be in commented form

Reply:
Done